extensions/amp-story-interactive/0.1/amp-story-interactive-slider.js

```diff
@@ -109,6 +109,14 @@ export class AmpStoryInteractiveSlider extends AmpStoryInteractive {
     this.inputEl_.addEventListener('change', () => {
       this.onRelease_();
     });
+
+    this.inputEl_.addEventListener(
+      'touchmove',
+      (event) => {
+        event.stopPropagation();
+      },
```

Comment:
Since this is one line, it can be written without brackets like:
`(event) => event.stopPropagation()`